### PR TITLE
neofetch: add pt_BR translation

### PR DESCRIPTION
**Pull request by andrecoco**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/common/neofetch.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Volte à configuração padrão, ou crie-a se for a primeira vez que o programa for executado:
+
+`neofetch`
+
+- Habilite/Desabilite uma linha de informação da saída, onde 'infoname' é o nome da função no arquivo de configuração, por exemplo, 'memory':
+
+`neofetch --{{enable|disable}} {{infoname}}`
+
+- Ocultar/Mostrar arquitetura do SO:
```
Comment (navarroaxel) [inline pages.pt_BR/common/neofetch.md:14]:
```suggestion
- Oculta/Mostra arquitetura do SO:
```

File: pages.pt_BR/common/neofetch.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`neofetch --os_arch {{on|off}}`
+
+- Habilitar/desabilitar a marca da CPU na saída:
```
Comment (navarroaxel) [inline pages.pt_BR/common/neofetch.md:18]:
```suggestion
- Habilita/Desabilita a marca da CPU na saída:
```

File: pages.pt_BR/common/neofetch.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# neofetch
+
+> Ferramenta CLI para exibir informações sobre seu sistema operacional, software e hardware.
+> Mais informações: <https://github.com/dylanaraps/neofetch>.
+
+- Volte à configuração padrão, ou crie-a se for a primeira vez que o programa for executado:
```
Comment (navarroaxel) [inline pages.pt_BR/common/neofetch.md:6]: Please presente indicativo
```suggestion
- Volta à configuração padrão, ou crie-a se for a primeira vez que o programa for executado:
```

File: pages.pt_BR/common/neofetch.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`neofetch`
+
+- Habilite/Desabilite uma linha de informação da saída, onde 'infoname' é o nome da função no arquivo de configuração, por exemplo, 'memory':
```
Comment (navarroaxel) [inline pages.pt_BR/common/neofetch.md:10]:
```suggestion
- Habilita/Desabilita uma linha de informação da saída, onde 'infoname' é o nome da função no arquivo de configuração, por exemplo, 'memory':
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]
Remember that you can apply all the suggestions in one commit, adding them to a batch <https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request>.
image

**Review by marchersimon — APPROVED** [review]
Thanks @andrecoco

**Pull request merged**